src/acceptanceTest/java/uk/gov/hmcts/futurehearings/hmi/acceptance/common/test/HmiCommonHeaderTest.java

```diff
@@ -238,60 +237,62 @@ void test_request_created_at_invalid_values(String requestCreatedAtKey, String r
+                                                   String requestCreatedAtVal) throws Exception {
         commonDelegate.test_expected_response_for_supplied_header(
                 getAuthorizationToken(),
-                getRelativeURL(), getInputPayloadFileName(),
+                getRelativeUrl(), getInputPayloadFileName(),
                 createHeaderWithRequestCreatedAtSystemValue(requestCreatedAtVal, sourceSystem),
                 null,
                 getUrlParams(),
                 getHttpMethod(),
                 getHttpSuccessStatus(),
                 getInputFileDirectory(),
                 getHmiSuccessVerifier(),
-                "The request was received successfully.",null);
+                "The request was received successfully.", null);
     }
 
     @Test
     @DisplayName("Accept System Header invalid value")
     void test_accept_at_with_invalid_value() throws Exception {
         commonDelegate.test_expected_response_for_supplied_header(
                 getAuthorizationToken(),
-                getRelativeURL(), getInputPayloadFileName(),
+                getRelativeUrl(), getInputPayloadFileName(),
                 createHeaderWithAcceptTypeAtSystemValue("application/pdf", sourceSystem),
                 null,
                 getUrlParams(),
                 getHttpMethod(),
                 HttpStatus.NOT_ACCEPTABLE,
                 getInputFileDirectory(),
                 getHmiErrorVerifier(),
-                "Missing/Invalid Media Type",null);
+                "Missing/Invalid Media Type", null);
     }
 
     @ParameterizedTest(name = "Duplicate System headers with valid values - Param : {0} --> {1}")
     @CsvSource(value = {
-            //System Headers of Accept and Content-Type could not be duplicated as Rest Assured seems to remove the Duplication of valid same values.
+            //System Headers of Accept and Content-Type could not be
+            // duplicated as Rest Assured seems to remove the Duplication of valid same values.
             //This should be tested manually using Postman.
-            "Source-System,NIL","Source-System,''","Source-System,CFT",
-            "Destination-System,NIL","Destination-System,''","Destination-System,SNL",
-            "Request-Created-At,NIL","Request-Created-At,''","Request-Created-At,2002-10-02T15:00:00Z"
-    }, nullValues = "NIL")
+            "Source-System,NIL", "Source-System,''", "Source-System,CFT",
+            "Destination-System,NIL", "Destination-System,''", "Destination-System,SNL",
+            "Request-Created-At,NIL", "Request-Created-At,''", "Request-Created-At,2002-10-02T15:00:00Z"
+        }, nullValues = "NIL")
     void test_duplicate_headers(String duplicateHeaderKey, String duplicateHeaderValue) throws Exception {
 
         final String expectedErrorMessage =
                         "Missing/Invalid Header " + duplicateHeaderKey;
         Map<String,String> duplicateHeaderField  = new HashMap<>();
```

Comment:
use ConcurrentHashMap instead of HashMap

Reply:
Done